legacy.go

```diff
@@ -141,6 +142,11 @@ func (app *App) RegisterLegacyModules() error {
 	transferStack = transfer.NewIBCModule(app.TransferKeeper)
 	transferStack = ratelimit.NewIBCMiddleware(app.RateLimitKeeper, transferStack)
 	transferStack = forwarding.NewMiddleware(transferStack, app.AccountKeeper, app.ForwardingKeeper)
+	transferStack = orbiter.NewIBCMiddleware(
```

Comment:
As you mentioned in Slack @0xstepit, I think this ordering is fine -- but would love @jtieri to confirm!